desktop/core/src/desktop/views.py

```diff
@@ -15,44 +15,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from future import standard_library
-
-standard_library.install_aliases()
-import json
-import logging
 import os
 import re
-import six
-import socket
 import sys
-import tempfile
+import json
 import time
-import traceback
+import socket
+import logging
 import zipfile
-import validate
+import tempfile
+import traceback
+from wsgiref.util import FileWrapper
 
-from django.http import HttpResponseRedirect
+import six
+import validate
+import django.views.debug
+from configobj import ConfigObj, ConfigObjError, get_extra_values
 from django.conf import settings
 from django.contrib.staticfiles.storage import staticfiles_storage
-from django.shortcuts import render as django_render
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseRedirect
 from django.http.response import StreamingHttpResponse
+from django.shortcuts import redirect, render as django_render
 from django.urls import reverse
-from django.shortcuts import redirect
 from django.views.decorators.http import require_POST
-from configobj import ConfigObj, get_extra_values, ConfigObjError
-from wsgiref.util import FileWrapper
+from future import standard_library
```

Comment:
Good to remove

Reply:
Removed line 41: from future import standard_library